desktop/ui/src/main/java/org/datacleaner/windows/AboutDialog.java

```diff
@@ -107,7 +107,7 @@ protected int getDialogWidth() {
 
     @Override
     protected boolean isWindowResizable() {
-        return true;
+        return false;
```

Comment:
Because it is stupid. It's not something you work with. You read it, you close it.